src/main/java/uk/gov/hmcts/reform/sscs/service/placeholders/PlaceholderService.java

```diff
@@ -70,12 +40,15 @@ public void build(SscsCaseData caseData, Map<String, Object> placeholders, Addre
         if (description == null && appeal.getBenefitType() != null && appeal.getBenefitType().getCode() != null) {
             description = Benefit.getBenefitOptionalByCode(appeal.getBenefitType().getCode()).map(Benefit::getDescription).orElse(StringUtils.EMPTY);
         }
+        String shouldHideNino = Benefit.CHILD_SUPPORT.getDescription().equals(description) ? YesNo.YES.getValue() : YesNo.NO.getValue();
```

Comment:
This is okay, but why not use appeal.getBenefitType().getCode() instead of the description to check to hide the nino?

Reply:
Done